### PR TITLE
[k8s-configuration] BREAKING CHANGE: `az k8s-configuration create`: Removal of create cmd for flux v1

### PR DESCRIPTION
**Pull request by bavneetsingh16**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally? (`pip install wheel==0.30.0` required)
- [ ] My extension version conforms to the [Extension version schema](https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md)

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
k8s-configuration

**Comment by AllyW** [comment]
> ❌Azure CLI Extensions Breaking Change Test
> > ❌k8s-configuration
> > 
> > 
> > 
> > rule
> > cmd_name
> > rule_message
> > suggest_message
> > 
> > 
> > 
> > 
> > ❌ [1002 - CmdRemove](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1002.md)
> > k8s-configuration create
> > cmd `k8s-configuration create` removed
> > please confirm cmd `k8s-configuration create` removed

breaking change in new release should increase the major version number to be `2.0.0`. For cli extension versioning shcema, please refer to [Extension version schema](https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md)

File: src/k8s-configuration/setup.py [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
     logger.warn("Wheel is not available, disabling bdist_wheel hook")
 
-VERSION = "1.7.0"
+VERSION = "1.8.0"
```
Comment (AllyW) [inline src/k8s-configuration/setup.py:19]: breaking change in new release should increase the major version number to be `2.0.0`. For cli extension versioning shcema, please refer to [Extension version schema](https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md)

**Review by yanzhudd — APPROVED** [review]

**Pull request merged**